objax/functional/loss.py

```diff
@@ -48,6 +52,25 @@ def cross_entropy_logits_sparse(logits: JaxArray, labels: Union[JaxArray, int])
     return logsumexp(logits, axis=1) - logits[jn.arange(logits.shape[0]), labels]
 
 
+def mean_squared_logarithmic_error(y_true: JaxArray,
+                                    y_pred: JaxArray,
+                                    eps: float = 2 ** -17) -> JaxArray:
+    """Computes the the mean squared logarithmic error.
```

Comment:
Is there a link to a paper or to wikipedia we could add here?

Reply:
I couldn't find this on wiki. But it is heavily used in `kaggle` competition as an evaluation metric. Here are the counterparts in `sklearn` and `TF` 
1. https://scikit-learn.org/stable/modules/model_evaluation.html#mean-squared-log-error
2. https://www.tensorflow.org/api_docs/python/tf/keras/losses/MSLE

Reply:
Thanks for the information. Well I guess there's no need to add anything then.